packages/start-devtools/src/core.tsx

```
@@ -0,0 +1,11 @@
+
+import { constructCoreClass } from '@tanstack/devtools-utils/solid'
+import { lazy } from 'solid-js'
+
+const Component = lazy(() => import('./StartDevtools'))
+
+export interface StartDevtoolsInit { }
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)